lib/main.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+
+import 'package:anxiety_go/components/carousal_slider.dart';
+
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'AnxietyGo',
+      debugShowCheckedModeBanner: false,
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // Try running your application with "flutter run". You'll see the
+        // application has a blue toolbar. Then, without quitting the app, try
+        // changing the primarySwatch below to Colors.green and then invoke
+        // "hot reload" (press "r" in the console where you ran "flutter run",
+        // or simply save your changes to "hot reload" in a Flutter IDE).
+        // Notice that the counter didn't reset back to zero; the application
+        // is not restarted.
+        primarySwatch: Colors.blue,
+      ),
+       home: const AppCarousel(),
+       //home: const MyHomePage(title: 'AnxietyGo'),
```

Comment:
Revert the changes on line 31

the home page should be MyHomePage(title: 'AnxietyGo')